packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -774,18 +774,40 @@ export const Webchat = forwardRef((props, ref) => {
   }
 
   const userInputEnabled = isUserInputEnabled()
-  const emojiPickerEnabled = getThemeProperty(
-    WEBCHAT.CUSTOM_PROPERTIES.enableEmojiPicker,
-    props.enableEmojiPicker
-  )
-  const attachmentsEnabled = getThemeProperty(
-    WEBCHAT.CUSTOM_PROPERTIES.enableAttachments,
-    props.enableAttachments
-  )
+
+  const isEmojiPickerEnabled = () => {
+    const hasCustomEmojiPicker = !!getThemeProperty(
```

Comment:
Is !! needed? Negating twice will leave the same value as if we don't have it.

Reply:
it would not be necessary but I do it because the function getThemeProperty does not return a boolean but I use the result as a boolean